bike_kollective/lib/data/model/user.dart

```diff
@@ -4,7 +4,9 @@ import 'package:bike_kollective/data/model/bk_document_reference.dart';
 // User data, format version 1
 class UserModel {
   final BKDocumentReference? docRef;
-  final String uid;
+  final String? uid;
+  final String? email;
+  final String? password;
```

Comment:
Usually we don't want to store raw password strings in the database. Is there a reason why auth can't store this info for us? My understanding is auth can store this info safely while the rest of the app deals exclusively with uid.